src/inmanta/agent/agent.py

```diff
@@ -1283,14 +1277,35 @@ async def get_code(
 
         return resource_install_specs, invalid_resource_types
 
+    def initialize_loader(self) -> None:
```

Comment:
I think we can move the entire functionality to `InProcessManager`, including all these bookkeeping attributes and the `_install` method. Wdyt?

Reply:
Yes, I agree :+1: